src/main/webapp/pages/projects/project_new.html

```diff
@@ -213,6 +220,25 @@ <h2 class="panel-title" th:text="#{projects.create.heading}">Create a New Projec
           }
         });
 
+        $("#useCartSamples").on("change", function(e) {
```

Comment:
Add a comment above these 2 functions about what they're doing please.

Reply:
Fixed in 9e258b0be30f71fb335b7baf9d4591ba3e44601b